packages/server/utils/s3.js

```diff
@@ -1,113 +0,0 @@
-import aws from 'aws-sdk'
-import promisify from 'es6-promisify'
-import mime from 'mime-types'
-import protocolRelativeUrl from './protocolRelativeUrl'
-
-/*
- * Initializing AWS S3 implicitly uses environment variables
- * AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
- */
-const s3 =
```

Comment:
creating things on server startup isn't great, now it gets created when the S3 manager is created